packages/codemirror-graphql/src/info.ts

```diff
@@ -62,9 +62,8 @@ CodeMirror.registerHelper(
       return;
     }
 
-    const state = token.state;
-    const kind = state.kind;
-    const step = state.step;
+    const { state } = token;
+    const { kind, step } = state;
```

Comment:
Could we even enforce this?
```suggestion
    const { state: { kind, step } } = token;
```

Reply:
I don't think :)
But if we force I would prefer to
`const { kind, step } = token.state;`

also for better destructuring there is https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/consistent-destructuring.md